src/cmt_encode_msgpack.c

```diff
@@ -144,6 +150,11 @@ static void pack_header(mpack_writer_t *writer, struct cmt *cmt, struct cmt_map
 
         mpack_finish_array(writer);
     }
+    else if (map->type == CMT_COUNTER){
+        /* aggregation_type */
+        mpack_write_cstr(writer, "agg_type");
```

Comment:
Could you please rename this to `aggregation_type`?

Reply:
It was for reducing memory footprint. Anyway I changed it. :)